crates/sui-framework/src/natives/address.rs

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::legacy_create_signer_cost;
+use move_binary_format::errors::PartialVMResult;
+use move_core_types::{account_address::AccountAddress, u256::U256};
+use move_vm_runtime::native_functions::NativeContext;
+use move_vm_types::{
+    loaded_data::runtime_types::Type, natives::function::NativeResult, pop_arg, values::Value,
+};
+use smallvec::smallvec;
+use std::collections::VecDeque;
+
+const E_ADDRESS_PARSE_ERROR: u64 = 0;
+
+const E_CANT_CONVERT_ADDRESS_TO_U256: u64 = 1;
+
+// Implementation of the Move native function address_from_bytes(bytes: vector<u8>): address;
```

Comment:
nit: comment needs updating.

```suggestion
// Implementation of the Move native function address::from_bytes(bytes: vector<u8>): address;
```